packages/rocketchat-ui/client/views/app/forwardMessage.html

```diff
@@ -0,0 +1,31 @@
+<template name="forwardMessage">
+	<section class="forward-message">
+		<header class="forward-message__header">
+			<h1 class="forward-message__title">{{_ "Forward_Message_Modal_Title"}}</h1>
+		</header>
+
+		<form class="forward-message__content">
+			{{#nrr nrrargs 'messageAttachment' attachment}}{{/nrr}}
+
+			<div class="forward-message__recipients">
+				<label>{{_ "Choose_a_room"}}</label><br>
```

Comment:
Since you can add multiple, perhaps it is a better idea to write "Choose rooms" (or recipients/destinations)?